nutanix/resource_nutanix_network_security_rule.go

```diff
@@ -1483,6 +1551,30 @@ func flattenNetworkRuleList(networkRules []*v3.NetworkRule) []map[string]interfa
 	return ruleList
 }
 
+func referenceListSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		Computed: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"kind": {
+					Type:     schema.TypeString,
+					Computed: true,
+				},
+				"uuid": {
```

Comment:
Shouldn't UUID be marked as optional & computed as it will be an input to API.

@jan-walther thoughts? 

Can you also attach a working example in the PR?